core/src/main/java/org/mule/runtime/core/policy/OperationPolicyPointcutParametersFactory.java

```diff
@@ -30,9 +30,10 @@
    *
    * @param operationIdentifier identifier of the message source
    * @param operationParameters set of parameters that are going to be used to execute the operation.
+   * @param flowName name of the flow executing the operation.
    * @return the pointcut parameters.
    */
   PolicyPointcutParameters createPolicyPointcutParameters(ComponentIdentifier operationIdentifier,
-                                                          Map<String, Object> operationParameters);
+                                                          Map<String, Object> operationParameters, String flowName);
```

Comment:
Looks like flowName should be non empty (ie: javadoc must be updated and validated properly validated)